sample/migrations/20230621162215_courses.sql

```diff
@@ -52,23 +51,28 @@ CREATE INDEX IF NOT EXISTS idx_course_offerings_course_id
 CREATE INDEX IF NOT EXISTS idx_course_offerings_year_term
     ON course_offerings (year, term);
 
--- Course Schedule Table
-CREATE TABLE IF NOT EXISTS course_schedule (
+-- Class Schedule Table
+CREATE TABLE IF NOT EXISTS class_schedule (
     id SERIAL PRIMARY KEY,
-    section VARCHAR(30) NOT NULL,
-    start_time TIME,
-    end_time TIME,
-    meeting_days VARCHAR(7) NOT NULL,
-    instructor_name VARCHAR(50) NOT NULL,
-    room_name VARCHAR(30) NOT NULL,
-    campus_name VARCHAR(50) NOT NULL,
+    class_section SMALLINT NOT NULL,
+    class_number SMALLINT NOT NULL,
+    component VARCHAR(10) NULL,
+    start_time TIMESTAMP,
+    end_time TIMESTAMP,
+    monday BOOLEAN NOT NULL,
+    tuesday BOOLEAN NOT NULL,
+    wednesday BOOLEAN NOT NULL,
+    thursday BOOLEAN NOT NULL,
+    friday BOOLEAN NOT NULL,
+    saturday BOOLEAN NOT NULL,
+    sunday BOOLEAN NOT NULL,
+    instructor_name TEXT NULL,
+    location VARCHAR(50) NULL,
     course_offering_id INTEGER NOT NULL,
     max_enrollment SMALLINT NOT NULL DEFAULT 0,
     current_enrollment SMALLINT NOT NULL DEFAULT 0,
-    CONSTRAINT fk_course_schedule_to_courses FOREIGN KEY (course_offering_id)
+    CONSTRAINT fk_class_schedule_to_courses FOREIGN KEY (course_offering_id)
         REFERENCES course_offerings (id)
         ON DELETE CASCADE,
-    CONSTRAINT chk_meeting_days CHECK (meeting_days ~ '^[YN]{7}$'),
-    CONSTRAINT chk_course_schedule_current_enrollment CHECK (current_enrollment <= max_enrollment),
-    CONSTRAINT chk_course_schedule_times CHECK (end_time > start_time)
+    CONSTRAINT chk_class_schedule_times CHECK (end_time >= start_time)
 );
```

Comment:
@Justin-Kwan I removed the `curr <= max` enrollment constraint because sections can have **overrides** resulting in more students than "max" capacity. I also set `end_time >= start_time` because some courses are "all-day" ex. online courses.